index.d.ts

```diff
@@ -2060,14 +2032,16 @@ declare namespace Shopify {
     requires_shipping: boolean;
     sku: string;
     title: string;
-    variant_id: number | null;
```

Comment:
`null` was added in https://github.com/MONEI/Shopify-api-node/commit/b3b34ccf4f92e8292096c8660793c47922599f83 I think it should not be removed.

Reply:
I relied on the documentation too much. It’s not mentioned at https://shopify.dev/docs/admin-api/rest/reference/orders/order#line-items-property-2020-01 but according to https://community.shopify.com/c/Shopify-APIs-SDKs/Orders-line-item-variant-id-fields-are-null/td-p/130401 yes, this can be `null`